### ISSUE TITLE
Enable a11y on mobile safari

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#333]
Now that iOS VoiceOver on safari is a supported platform for a11y, it should be enabled on that platform.

**Comment by jessegreenberg** [comment]
Done, its working great.
